### PR TITLE
Update section 'Create a Plone theme python package'

### PR DESCRIPTION
**Pull request by staeff**
Fixes:
- configure.zcml generated by mr.bob is not sufficient to use the package without problems and needs to be adjusted (765df04)
- less files need to be renamed for successful resource compilation (44b25b4)

Improves:
- Update outdated text and fix some issues

### CODE REVIEW

**Comment by pbauer** [comment]
@cdw9 @MrTango can you please review and merge this?

File: theming/theme-package.rst [hunk 1]
```diff
@@ -91,6 +91,13 @@ Deactivate mrbob virtualenv:
 
    (mrbobvenv)maik@planetmobile:~/develop/plone/plonetheme.tango$ deactivate
 
+.. note::
+
+  Currently you need make a change to need to make some changes to
+  src/plonetheme/tango/configure.zcml. Please copy over the contents
+  of configure.zcml from the collective/plonethem.tango - repo:
```
Comment (gforcada) [inline theming/theme-package.rst]: plonetheme.tango maybe? It is missing an *e*

**Comment by MrTango** [comment]
Hi sorry, I didn't see it before. Im now in the middle of refacoring the same part. When I'm done with it I'll merge what is left.

**Comment by MrTango** [comment]
I'll close this, because it's outdated after my refactoring of the training.

**Pull request closed without merging**